# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/xforms/hotfix-fno-common.patch
# Copyright (C) 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

Make the per spec form curobj static like in the other sp_*.c files,
fixes the link with -fno-common.

--- a/fdesign/sp_spinner.c	2013-12-14 14:26:25.000000000 +0100
+++ b/fdesign/sp_spinner.c	2026-09-26 17:46:48.060339967 +0200
@@ -29,7 +29,7 @@
 #include "spec/spinner_spec.h"
 
 static FD_spinnerattrib * spn_attrib;
-FL_OBJECT *curobj;
+static FL_OBJECT *curobj;
 
 
 /***************************************
--- a/fdesign/sp_twheel.c	2014-06-28 22:34:29.000000000 +0200
+++ b/fdesign/sp_twheel.c	2026-09-26 17:46:48.058688708 +0200
@@ -38,7 +38,7 @@
 #include "spec/twheel_spec.h"
 
 static FD_twheelattrib * twheel_attrib;
-FL_OBJECT * curobj;
+static FL_OBJECT * curobj;
 
 
 /***************************************
